mitosheet/src/mito/Mito.tsx

```diff
@@ -335,6 +335,23 @@ export const Mito = (props: MitoProps): JSX.Element => {
         loadPlotly()
     }, [])
 
+    /**
+     * This useEffect handles when the user creates a graph, then presses undo.
+     */
+    useEffect(() => {
+        if (uiState.currOpenTaskpane.type === TaskpaneType.GRAPH &&
+            analysisData.graphDataArray.length === 0) {
```

Comment:
How does this not get triggered when the graph taskpane I open but the graph has not been created yet?

Reply:
It'd be triggered once the `graphDataArray` changes, but at that point the length of the graphDataArray is actually >0.